views/tx.tmpl

```diff
@@ -47,7 +48,7 @@
                         {{else}}
                             <a href="/explorer/block/{{$.Block.BlockHash}}">{{$.Block.BlockHeight}}</a></td>
                         {{end}}
-
+                        <td>{{$.Type}}</td>
```

Comment:
A new column in the summary table is fine for now, but I think we need the tx type to be more conspicuous.  Something like this:

![image](https://user-images.githubusercontent.com/9373513/30403432-75ae96b4-9896-11e7-865a-e64fb0fe17b7.png)

Reply:
yes, will be addressed later